help/develop/getting-started.md

```diff
@@ -3,208 +3,215 @@
-
-5. We host regular Datagrok User Meetings. Ask Andrew or Dan for a Zoom link
-   (for example, by a private message at [Community Forum](https://community.datagrok.ai/)).
+Check out the following spaces for more information about Datagrok:
 
-## Datagrok Video Walkthrough
+1. [Community Forum]. This is where we post updates about the Datagrok platform and ecosystem. We recommend that you
+   subscribe at least to the following three topics on the forum:
+   * [JavaScript API Updates]
+   * [Visualization-related Updates]
+   * [Cheminformatics Updates]
 
-Let's get it started!
+2. [Datagrok Release History] &mdash; this page aggregates the release notes. For every new Datagrok build, the release notes
+   are automatically generated. However, for the major releases, we curate our release notes, which you can see on
+   [our GitHub page]. There, we detail the highlights, major items, improvements, and bug fixes in the releases. We also
+   provide links to the relevant [Community Forum] topics, examples, and sources.
 
-From our detailed range of [recorded user meetings and tutorials](https://www.youtube.com/channel/UCXPHEjOd4gyZ6m6Ji-iOBYg)
-we've curated these which let you start fast with building on Datagrok. Besides getting an overall idea about the platform,
-which will help you a lot all the way further in your work, in these videos you'll be directly introduced to our
-JavaScript API and platform extensions.
+3. [JS API Samples]. We add diverse examples of our JavaScript API to the Datagrok public website. You can use the
+   built-in JS Fiddler inside Datagrok to rapidly prototype interfaces and functions that will later become parts of
+   your packages.
 
-There's about 5 hours of watching to begin with. It's up to your taste if you watch everything first and then
-proceed to [exercises](#datagrok-tutorials-and-exercises), or prefer combining them on the way.
+   The sources of these API samples are stored in the [Datagrok ApiSamples package on GitHub].
 
-1. Start with a [Coffee Company](https://www.youtube.com/watch?v=tVwpRB8fikQ) Feature Demo to get an overall idea
+4. [Datagrok public repository]. We're passionate about making more parts of our work open-source, and we strive to
+   involve other developers into expanding the Datagrok platform.
 
-2. Going further, proceed with all videos from [the Platform Overview](video-contents.md/#getting-started)
+   The Datagrok public repository is not only a great source of API examples, but also a repository of fine-crafted
+   patterns, such as a [ChEMBL data repository browser], and plugins, such as [custom cell renderers], [custom viewers],
+   [custom filters], and others.
 
-3. Using Datagrok JS API would become your regular routine. Proceed with all videos [the JS API Overview](video-contents.md/#java-script-api)
+   Finally, you can also find our packages for domains such as [Digital Signal Processing], [Biosignals], Cheminformatics,
```

Comment:
@dskatov could you find a link for Cheminformatics, please? I see two options here:
- https://github.com/datagrok-ai/public/tree/master/packages/Chem
- https://github.com/datagrok-ai/public/tree/master/packages/ChemScripts